src/Microsoft.AspNetCore.Razor.Language/IRazorImportFeature.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    public interface IRazorImportFeature : IRazorProjectEngineFeature
```

Comment:
In a future PR i'll introduce `IRazorImportExpander` to enable multiple import expanders to take part in import discovery. That will be an implementation detail of this.

Reply:
I think you've lost me on the to-do here. Wouldn't MVC just supply its own implementation of this? WOuldn't the default implemenntation just noop?